assets/settings/default.json

```diff
@@ -679,6 +679,14 @@
     "**/.classpath",
     "**/.settings"
   ],
+  // Add files or globs of files that will be included by Zed, even when
+  // ignored by git. This is useful for files that are not tracked by git,
+  // but are still important to your project. Note that globs that are
+  // overly broad can slow down Zed's file scanning. Overridden by `file_scan_exclusions`.
+  "file_scan_inclusions": [
+    ".env*",
+    "docker-compose.*.yml"
```

Comment:
Was intentional to include in the default settings?
I can understand `.env*`, but `docker-compose.*.yml` seems pretty niche.

Reply:
I think this was something that I wanted to be part of the discussion but it ended up being forgotten. We can definitely remove the `docker-compose.*.yml`. We should definitely try to keep this as minimal as possible to avoid adding any unnecessary and unexpected file scanning.

Reply:
Fixed. Thanks.